### PR TITLE
Setup + verizon validate device + action generator

### PR DESCRIPTION
**Pull request by abulrim**

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
```
Comment (pedromartinez) [inline bin/console]: As it turns out, so long as conduit-telispire.rb has all of those requires, `bundle console` will do the same thing.

File: bin/console [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+require "bundler"
```
Comment (coreypurcell) [inline bin/console]: you don't really need to create a bin/console. `bundle console` should be able to do this

**Comment by coreypurcell** [comment]
looks really nice. Really like the generator with thor. If you think it's worth keeping bin/console as a way to document that `bundle console` works, I could 🆗 that.

File: lib/conduit/telispire/actions/verizon_validate_device.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+module Conduit::Driver::Telispire
+  # Validate a verizon device based on its `esn`
+  class VerizonValidateDevice < Conduit::Driver::Telispire::Base
+    operation           :verizon_validate_device
+
+    # Required attributes
+    #
+    # * `esn` {String} esn of the device
+    # * `is4G` {Boolean}
+    required_attributes :esn, :is4G
```
Comment (pedromartinez) [inline lib/conduit/telispire/actions/verizon_validate_device.rb]: Right now, the Connect codebase expects all these drivers to have a similar interface. We can change that, but it'll be work to do it. The easiest way is to name the action and attributes the same as conduit-sprint has them.

https://github.com/conduit/conduit-sprint/blob/master/lib/conduit/sprint/actions/query_device_info.rb

@georgeismike fyi

Comment (abulrim) [inline lib/conduit/telispire/actions/verizon_validate_device.rb]: Hm... In this case the two APIs expect different device identifiers (`esn` vs `meid`)

Comment (pedromartinez) [inline lib/conduit/telispire/actions/verizon_validate_device.rb]: Actually, esn and meid are the same thing and can be used interchangeably.

**Comment by pedromartinez** [comment]
Looks good, just had a couple comments.

File: lib/conduit/telispire/actions/verizon_validate_device.rb (hunk 3 above)
Comment (abulrim) [inline lib/conduit/telispire/actions/verizon_validate_device.rb]: 👍

**Comment by abulrim** [comment]
Updated

File: lib/conduit/telispire/actions/query_device_info.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+module Conduit::Driver::Telispire
+  # Validate a verizon device based on its `meid`
+  class QueryDeviceInfo < Conduit::Driver::Telispire::Base
+    operation           :verizon_validate_device
+
+    # Required attributes
+    #
+    # * `meid` {String} meid of the device
+    # * `is4G` {Boolean}
+    required_attributes :meid, :is4G
```
Comment (pedromartinez) [inline lib/conduit/telispire/actions/query_device_info.rb]: What does is4G do?

Comment (abulrim) [inline lib/conduit/telispire/actions/query_device_info.rb]: forgot to remove it

**Comment by abulrim** [comment]
Removed `is4G`

**Comment by pedromartinez** [comment]
👍

**Pull request merged**